src/qt/buynamespage.cpp

```diff
@@ -9,10 +9,14 @@
 #include <qt/platformstyle.h>
 #include <qt/walletmodel.h>
 #include <rpc/protocol.h>
-
+#include <names/applications.h>
 #include <names/encoding.h>
 #include <univalue.h>
 
+#include <string>
+#include <algorithm>
+#include <exception>
```

Comment:
What are we using from these three headers?

Reply:
(To be clear, I'm not saying they are definitely superfluous, I'm just asking for info.)